### PR TITLE
fix(aqua-ipfs): Update example

### PR DESCRIPTION
**Pull request by InversionSpaces**

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -13,16 +14,15 @@
...
+    "ipfs-http-client": "^60.0.0",
+    "@multiformats/multiaddr": "^12.1.1",
     "it-all": "^1.0.5",
     "uint8arrays": "^2.1.5"
   },
   "devDependencies": {
     "typescript": "^4.0.0"
   }
-}
+}
```
Comment (InversionSpaces) [inline example/package.json:28]:
```suggestion
}

```

File: example/README.md [hunk 2]
```diff
@@ -1,8 +1,7 @@
...
+While in `example` directory, `npm run start` will run `index.ts` against Fluence TestNet
 
 # Run example locally
-1. Go to `local-network` 
-2. Run `docker compose up -d` to start Fluence cluster of 3 nodes
-3. Go back to `../example`
-4. Run `npm run start:local`
+To run example locally:
+1. Spin up local environment through [Fluence CLI](https://github.com/fluencelabs/cli)'s `local` command
+2. Pass local peers multiaddresses to `main` in `index.ts`
```
Comment (folex) [inline example/README.md:7]: Do you mean "change `const IPFS_MULTIADDR`"?

**Review by folex — APPROVED** [review]

File: example/README.md (hunk 2 above)
Comment (InversionSpaces) [reply example/README.md]: No, I meant `environment: Relay[]` parameter of `main` function

File: example/index.ts [hunk 3]
```diff
@@ -23,11 +23,11 @@ import all from "it-all";
 import uint8ArrayConcat from "uint8arrays/concat.js";
 
 // Multi address of the IPFS node
-// we will work with in Fluence Network
+// we will work with through the Fluence Network
 const IPFS_MULTIADDR = multiaddr("/dns4/ipfs.fluence.dev/tcp/5001");
 
 /**
- * @param environment - array of network nodes (two are needed)
+ * @param environment - array of fluence network nodes (two are needed)
```
Comment (boneyard93501) [inline example/index.ts:30]: Why Fluence lower caps?

**Pull request merged**